app/code/Magento/Sales/Model/Order/Item.php

```diff
@@ -724,6 +724,7 @@ public function getAmountRefunded()
 
     /**
      * Returns applied_rule_ids
+     * Rules are comma separated if there are more than one.
```

Comment:
Please include a blank line before or add the "(comma separated)" information in return line, removing this one.